src/domain/__tests__/ConsumeEventStreamRecord.tests.ts

```diff
@@ -0,0 +1,48 @@
+import { makeConsumeEventStreamRecord } from '../ConsumeEventStreamRecord';
```

Comment:
If we don't want to repeat every time the invocation of the test, we can evaluate to use `jest-each` and write tests like this:
```
describe('makeConsumeEventStreamRecord', () => {
  each([
    {
      title: 'should return an empty array',
      input: data.consumeEventStreamRecord.input,
      records: [],
      expected: {
        statusCode: 200,
        returned: [],
      },
    },
    {
      title: 'should return an event',
      input: data.consumeEventStreamRecord.input,
      records: [data.newNotificationRecord],
      expected: {
        statusCode: 200,
        returned: data.consumeEventStreamResponse.returned,
      },
    },
...
  ]).it('$title', ({ input, records, expected }) => {
    const actual = makeConsumeEventStreamRecord(data.makeTestSystemEnv())(input)(records);
    expect(actual.output.statusCode).toStrictEqual(expected.statusCode);
    expect(actual.output.returned).toStrictEqual(expected.returned);
  });
});
```

Reply:
Sincerely I'm not a big fun of `each`, the types are erased.

Reply:
That's true! I see an advantage to have only one place where add the new tests (in the each part), but the drawback is what you said.